solutions/src/test/java/ru/sberbank/school/task13/Employee.java

```diff
@@ -0,0 +1,35 @@
+package ru.sberbank.school.task13;
+
+
+/**
+ * Created by Mart
+ * 20.07.2019
+ **/
+public class Employee {
+    private int age;
+    private String name;
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return "Employee{" +
```

Comment:
&#39;+&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a140148449>*